src/Protocol/Diaspora.php

```diff
@@ -2351,8 +2379,9 @@ private static function receiveReshare(array $importer, SimpleXMLElement $data,
 			return true;
 		}
 
-		$original_person = FContact::getByURL($root_author);
-		if (!$original_person) {
+		try {
+			$original_person = DI::dsprContact()->getByAddr($root_author);
+		} catch (HTTPException\NotFoundException $e) {
 			return false;
```

Comment:
Logging?

Reply:
There was none before, not sure it would be helpful.